Source/Moya.swift

```diff
@@ -21,7 +21,33 @@ public protocol TargetType {
     var path: String { get }
     var method: Moya.Method { get }
     var parameters: [String: AnyObject]? { get }
+    var parameterEncoding: ParameterEncoding { get }
     var sampleData: NSData { get }
+
+    func toEndpoint() -> Endpoint
+}
+
+public extension TargetType {
+    func toEndpoint() -> Endpoint {
+        let url = self.baseURL.URLByAppendingPathComponent(self.path).absoluteString
+        return Endpoint(URL: url, sampleResponseClosure: {.NetworkResponse(200, self.sampleData)}, method: self.method, parameters: self.parameters, parameterEncoding: parameterEncoding)
+    }
+}
+
+public extension TargetType {
+    var parameterEncoding: ParameterEncoding { return .URL }
+    var sampleData: NSData { return "".dataUsingEncoding(NSUTF8StringEncoding)! }
```

Comment:
Same problem as above here, though I really don't like the idea of allowing developers to bypass this anyway. Moya has a strong opinion on unit testing: it's a good idea and everyone should do it. If someone wants to add something like this to their `TargetType` type, that's fine, but they should feel bad about it :stuck_out_tongue: